Apps/Src/main.c

```diff
@@ -175,10 +187,5 @@ void Task_Init(void *p_arg){
     );
     assertOSError(err);
 
-
-    while(1){
-        Contactors_Set(MOTOR_CONTACTOR, Minions_Read(IGN_2), true); //turn on the contactor if the ign switch lets us
-        assertOSError(err);
-        OSTimeDlyHMSM(0, 0, 0, IGN_CONT_PERIOD, OS_OPT_TIME_HMSM_NON_STRICT, &err);
-    }
+    OSTaskDel(NULL, &err);
```

Comment:
This will delete the currently running task.
![image](https://github.com/lhr-solar/Controls/assets/22435311/7ae974fe-9cdb-42ff-9cef-602dc580a3ac)
If the currently running task is deleted then code after this line will not run.

Reply:
Got it. Documentation had mentioned that err needed to be in a different place, so I had assumed it could be checked inside the task but this makes sense. Thanks, for clarifying this!